tool/jt.rb

```diff
@@ -1134,7 +1134,8 @@ def test_specs(command, *args)
 
       options += %w[--excl-tag graalvm]
       options << '-t' << ENV['AOT_BIN']
-      options << '-T-XX:OldGenerationSize=2G'
+      options << '-T-XX:YoungGenerationSize=2G'
+      options << '-T-XX:OldGenerationSize=4G'
```

Comment:
Is 1G/2G like JS less optimal for specs?
2G of YoungGenerationSize will cause rather long young collections I would guess,